src/jit/codegenarm64.cpp

```diff
@@ -2616,9 +2616,15 @@ CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
                 if (targetType == TYP_STRUCT)
                 {
                     // At this point any TYP_STRUCT LclVar must be a two register argument
-                    assert(varDsc->lvSize() == 16); 
-                    emit->emitIns_R_S(ins_Load(TYP_I_IMPL), emitTypeSize(TYP_I_IMPL), targetReg, varNum, 0);
-                    emit->emitIns_R_S(ins_Load(TYP_I_IMPL), emitTypeSize(TYP_I_IMPL), REG_NEXT(targetReg), varNum, TARGET_POINTER_SIZE);
+                    assert(varDsc->lvSize() == 2*TARGET_POINTER_SIZE);
```

Comment:
In lowerarm64.cpp i see lcl var reg requirements are specified with srcCount=1.  Shouldn't the srcCount be 2?  Also, is genProduceReg() aware that GT_LCL_VAR has two regs assigned to it?

If it is not currently handled and we intend to work on it, at least we should capture  the work required with a TODO comment and a tracking git hub issue.